libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushNotificationInterface.java

```diff
@@ -37,5 +37,5 @@
  */
 public interface PushNotificationInterface {
 
-	void onPushMessageReceived(Bundle message);
+	void onPushMessageReceived(RemoteMessage message);
```

Comment:
FCM returns a much richer object for an incoming push notification - see [here](https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/RemoteMessage).